server/utils/risksAndNeeds/learningNeedsUtils.ts

```diff
@@ -0,0 +1,41 @@
+import ShowRisksAndNeedsUtils from '../referrals/showRisksAndNeedsUtils'
+import type { LearningNeeds } from '@accredited-programmes/models'
+import type { GovukFrontendSummaryListRowWithKeyAndValue } from '@accredited-programmes/ui'
+
+export default class LearningNeedsUtils {
```

Comment:
I'm sort of wondering whether all of this should be `LearningNeedsScore`/`learningNeedsScore`/`learning-needs-score` given the text in the navigation, but score is just one section of the actual page, and the endpoint and entity are just learning needs. Might ask design about the nav